docs/api/api/api.txt

```diff
@@ -1648,6 +1648,13 @@ GET /published/<project>/<repository>
 XmlResult: directory
 
 
+GET /published/<project>/<repository>?view=status
+
+  Delivers last publish information like time and repository build id
+
+XmlResult: publish_status
```

Comment:
Did you forget to commit this file?

Reply:
no, I was too lazy to create it. reference to buildstatus existed also before without the definition (like many others)

Reply:
ok